zilliqa/src/db.rs

```diff
@@ -361,13 +361,8 @@ impl Db {
 
         // Decompress file and write to temp file
         let input_filename = path.as_ref();
-        let temp_filename = input_filename.with_extension("part");
-        decompress_file(input_filename, &temp_filename)?;
-
-        // Read decompressed file
-        let input = File::open(&temp_filename)?;
-
-        let mut reader = BufReader::with_capacity(128 * 1024 * 1024, input); // 128 MiB read chunks
+        let buf_reader: BufReader<File> = BufReader::new(File::open(input_filename)?);
```

Comment:
I recall @bzawisto deliberately changing the buffer size to 128 MiB at some point in the past. Any reason not to keep it?
```suggestion
        let buf_reader = BufReader::with_capacity(128 * 1024 * 1024, File::open(input_filename)?); // 128 MiB read chunks
```

Reply:
No, though I don't think it's likely to particularly change anything any longer - a big BufReader probably helps when writing the result to disk but I'm not sure it helps much when reading, since the buffer cache will look ahead in any event. I did try to measure it, but restoring a mainnet checkpoint takes so long I gave up ... harmless, anyway, so done :-)